website/docs/introduction.md

```diff
@@ -7,22 +7,23 @@ slug: /
 
 ## Disclaimer
 
-It has been a year since we made the first **alpha release** of Docusaurus 2 and things have been pretty stable since then. Many of Facebook's new open source websites are using Docusaurus 2 now. At this point, we highly encourage you to use Docusaurus 2 over Docusaurus 1 for your new websites. For feedback and questions, chat with us on [**Discord**](https://discordapp.com/invite/docusaurus) :wink:.
+Docusaurus v2 is **still alpha** (since mid-2019) but already quite stable.
 
-**You should use this if:**
+We highly encourage you to **use Docusaurus v2 over Docusaurus v1**.
 
-- :white_check_mark: You want to contribute to Docusaurus 2
-- :white_check_mark: You want to build a modern website with client-side routing and prerendering
-- :white_check_mark: You want to provide feedback to make sure Docusaurus 2 suits your needs
-- :white_check_mark: You are curious about what comes next as a Docusaurus user
-- :white_check_mark: You want to ease the pain of migration in future
+Most users are already using v2 ([trends](https://www.npmtrends.com/docusaurus-vs-@docusaurus/core)), including [ReactNative](https://reactnative.dev), [Redux](https://redux.js.org/) and [many others](/showcase).
+
+**Use **Docusaurus v2** if:**
+
+- :white_check_mark: You want a modern Jamstack documentation site
+- :white_check_mark: You want a Single-Page-Application with client-side routing
```

Comment:
> with client-side routing

It seems to me that we need to indicate that we have SSR or prerendering.

Reply:
for me this is implied by Jamstack already